OutOfSchool/OutOfSchool.WebApi/Extensions/MappingExtensions.cs

```diff
@@ -45,6 +45,13 @@ public static CategoryDTO ToModel(this Category category)
             return categoryDto;
         }
 
+        public static ParentDTO ToModel(this Parent parent)
+        {
+            var parentDto =
+                Mapper<Parent, ParentDTO>(parent, cfg => { cfg.CreateMap<Parent, ParentDTO>(); });
+            return parentDto;
```

Comment:
can be just `return Mapper<Parent, ParentDTO>(parent, cfg => { cfg.CreateMap<Parent, ParentDTO>(); });`